components/admin/LayoutCard.tsx

```diff
@@ -34,24 +38,33 @@ export default function PermitHolderInfoCard(props: PermitHolderInfoCardProps) {
       borderRadius="8px"
       {...props}
     >
-      <Flex w="100%" justifyContent="flex-start" alignItems="center" mb={divider ? '20px' : '12px'}>
-        {typeof header === 'string' ? (
-          <Text as="h5" textStyle="display-small-semibold">
-            {header}
-          </Text>
-        ) : (
-          <>{header}</>
-        )}
-        {updated && (
-          <Text textStyle="caption" opacity="0.5" ml="12px">
-            updated
-          </Text>
-        )}
-        <Spacer />
-        {editModal}
-      </Flex>
-      {divider && <Divider mb="20px" />}
-      {children}
+      {!loading && (
+        <>
+          <Flex
+            w="100%"
+            justifyContent="flex-start"
+            alignItems="center"
+            mb={divider ? '20px' : '12px'}
+          >
+            {typeof header === 'string' ? (
+              <Text as="h5" textStyle="display-small-semibold">
+                {header}
+              </Text>
+            ) : (
+              <>{header}</>
+            )}
+            {updated && (
+              <Text textStyle="caption" opacity="0.5" ml="12px">
+                updated
+              </Text>
+            )}
+            <Spacer />
+            {editModal}
+          </Flex>
+          {divider && <Divider mb="20px" />}
+        </>
+      )}
+      {loading ? <LoadingSpinner /> : <>{children}</>}
```

Comment:
Put `children` in the same React fragment as above and make it a single ternary operator